### ISSUE TITLE
Changing the 'Viewed' saved fight breaks display

### ISSUE THREAD

**Issue by Diorik** [lunyx/CasualMeter#102]
[image](https://cloud.githubusercontent.com/assets/11672296/16538994/754a7ee0-3ffa-11e6-836b-ea6d48a5c71b.png)

As in picture, when selecting a different saved fight, the view breaks and all other participants are hidden. This was taken while alt tabbed from Tera, but with same character active.

**Comment by lunyx** [comment]
Do you have any way of reliably reproducing this by any chance?  I haven't come across this one before.

**Comment by Diorik** [comment]
100% repro. I've never had it not happen. I use the LFD tool, if that has any bearing. Next time I'm I'll try and get more details, whether this can happen before the instance is over or if only after, and see if running a non LFD group avoids it.

It does fix itself immediately after entering a new LFD group and starting to collect data.

**Comment by Diorik** [comment]
Happens even while the dungeon that I used to LFD to get into is still active, fixes upon collecting new data, but means I can never view old data without just pasting it.

**Comment by Diorik** [comment]
Win 7 Pro x64, 12gb RAM, GTX 980 Ti w/ 3 monitors, Core i7-2600k.

If you can give me a good idea where to start I'll try to take some time during the week to debug it.

**Comment by lunyx** [comment]
I believe this is the same as another issue that I recently fixed.
